entry.go

```diff
@@ -197,27 +210,167 @@ func (te *TunaEntry) StartReverse(stream *smux.Stream) error {
 		te.close()
 		return err
 	}
+	session, err := te.getSession(false)
+
+	bytesIn := make([]uint64, 256)
+	bytesOut := make([]uint64, 256)
+
+	claimInterval := time.Duration(te.config.ClaimInterval) * time.Second
+	onErr := nknsdk.NewOnError(1, nil)
+	var npc *nknsdk.NanoPayClaimer
+	lastComputed := common.Fixed64(0)
+	lastClaimed := common.Fixed64(0)
+	lastUpdate := time.Now()
+	isClosed := false
+
+	npc, err = te.wallet.NewNanoPayClaimer(int32(claimInterval/time.Millisecond), onErr, te.config.BeneficiaryAddr)
+	if err != nil {
+		log.Fatalln(err)
+	}
 
 	go func() {
-		session, err := te.getSession(false)
-		if err != nil {
-			return
-		}
-		stream, err := session.OpenStream()
```

Comment:
This open and close stream is needed to prevent TCP connection to be considered as idle and closed if there is no traffic within timeout